internal/slicer/slicer.go

```diff
@@ -36,21 +36,15 @@ func Run(options *RunOptions) error {
 		if path[0] != '/' {
 			panic("bug: tried to add relative path to known paths")
 		}
-		cleanPath := filepath.Clean(path)
-		slashPath := cleanPath
-		if path[len(path)-1] == '/' && cleanPath != "/" {
-			slashPath += "/"
-		}
+		path = fsutil.Clean(path)
```

Comment:
Could the path variable from before it is cleaned be called something like rawPath to make it clear that it has not been cleaned? And then you can refer to it simply as path from thereon out.

Reply:
Done.